class ExchangeError(Exception):
    pass


class OrderError(ExchangeError):
    pass
